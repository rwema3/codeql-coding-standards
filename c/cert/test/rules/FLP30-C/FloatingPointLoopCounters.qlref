rules/FLP30-C/FloatingPointLoopCounters.ql