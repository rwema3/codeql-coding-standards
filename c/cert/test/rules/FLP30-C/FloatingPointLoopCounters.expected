| test.c:3:3:4:3 | for(...;...;...) ... | Loop using a $@ of type floating-point. | test.c:2:9:2:9 | f | loop counter |
| test.c:5:3:7:3 | while (...) ... | Loop using a $@ of type floating-point. | test.c:2:9:2:9 | f | loop counter |
| test.c:9:3:11:22 | do (...) ... | Loop using a $@ of type floating-point. | test.c:2:9:2:9 | f | loop counter |
